src/firefly/js/visualize/ui/ImageSelectPanelResult.js

```diff
@@ -122,7 +122,7 @@ function imagePlotOnSurvey(crtCatalogId, request) {
             break;
 
         case TWOMASS:
-            wpr = WebPlotRequest.make2MASSRequest(wp, survey, sizeInDeg);
+            wpr = WebPlotRequest.make2MASSRequest(wp, survey, band, sizeInDeg);
```

Comment:
This is fine, just worth noting,  I think `ImageSelectPanelResult.js` is deprecated.